src/utils/index.ts

```diff
@@ -415,18 +415,26 @@ export const hasSameMembers = <T>(a: T[], b: T[]): boolean => {
   }
   return true;
 };
-export const isFriend = (user: User): boolean => !!(user.friendDiscoveryKey || user.friendName);
+export const isFriend = (user: Nullable<User>): boolean => {
+  if (!user) return false;
+  return !!(user.friendDiscoveryKey || user.friendName);
+};
 
 export const filterMessageListParams = (
   params: MessageListParams,
   message: SendableMessageType,
 ): boolean => {
+  const {
+    customTypesFilter = [],
+  } = params as MessageListParams & {
+    customTypesFilter: Array<string>;
+  };
```

Comment:
Does this look okay?
@bang9

Reply:
I believe we don't need `as MessageListParams & {
    customTypesFilter: Array<string>;
  };`

Reply:
Oh, that's true, thanks!